docs/01-get-started/loading-data-from-unified-graphql-schema.mdx

```diff
@@ -0,0 +1,135 @@
+  managing data through GraphQL, demonstrating the setup, execution, and
+  optimization of data requests. Understand the synergistic relationship between
+  Front-Commerce and Remix, with a focus on leveraging GraphQL for efficient
+  data handling and page rendering.
+sidebar_position: 4
+---
+
+<p>{frontMatter.description}</p>
+
+## Prerequisites
+
+Before diving into the code, ensure you have the following prerequisites in
+place:
+
+- A Front-Commerce project configured and ready to use
+- Basic knowledge of
+  [Remix routes](https://remix.run/docs/en/main/discussion/routes)
+- [FAQ GraphQL Module](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/tree/c317ab057ae0927e294a5287eb8e0be73ca7e1fb/skeleton/extensions/faq-demo/graphql)
+  installed and configured
+
+## Loading Data
+
+In your app directory, create a new route file (e.g.,
+`app/routes/_main.faq.$slug.tsx`) and add the following code:
+
+```tsx title="app/routes/_main.faq.$slug.tsx"
+import { json } from "@front-commerce/remix/node";
+import { useLoaderData } from "@front-commerce/remix/react";
+import type { LoaderFunctionArgs } from "@remix-run/node";
+import FaqDetail from "theme/pages/FaqDetail";
+import { isRouteErrorResponse, useRouteError } from "@remix-run/react";
+import { FaqEntryQueryDocument } from "~/graphql/graphql";
+import { FrontCommerceApp } from "@front-commerce/remix";
+
+// The route exports a `loader` function that is responsible
+// for fetching data and throwing errors, ensuring that
+// the main component is only rendered in the "happy path".
+export const loader = async ({ context, params }: LoaderFunctionArgs) => {
+  const { slug } = params;
+
+  // The `loader` uses the `FrontCommerceApp` class to access the Front-Commerce
+  // core from Remix. In this example, we use it to fetch data from the GraphQL
+  // unified API (similar to the one you're used to).
+  const app = new FrontCommerceApp(context.frontCommerce);
+  const response = await app.graphql.query(FaqEntryQueryDocument, {
+    input: { slug },
+  });
+  if (!response.faqEntry) {
+    throw new Response("Question not found", { status: 404 });
+  }
+
+  return json({
+    faqEntry: response.faqEntry,
+  });
+};
+
+// The route exports a `action` function that is responsible
+// for handling POST requests. It can be used to handle form
+// submissions.
+export const action = async ({
+  context,
+  params,
+  request,
+}: ActionFunctionArgs) => {
+  const { slug } = params;
+
+  const formData = await request.formData();
+  const app = new FrontCommerceApp(context.frontCommerce);
+  const response = await app.graphql.mutate(VoteIsFaqUsefulDocument, {
+    input: {
+      slug,
+      isFaqUseful: formData.get("isFaqUseful") === "true",
+    },
+  });
+  if (!response.voteIsFaqUseful) {
+    throw new Response("Question not found", { status: 404 });
+  }
+
+  return json({
+    success: response.voteIsFaqUseful?.success ?? false,
+  });
+};
```

Comment:
**issue:** this is part of the "Loading data" section which is incorrect

**suggestion:** create another "Mutate data" section explaining that same concepts seen for loading data can be used for mutating data